README.md

```diff
@@ -12,7 +12,6 @@ The ProteoGenomics database generation workflow (**pgdb**) use the [pypgatk](htt
 
 ## Introduction
 
-<!-- TODO nf-core: Write a 1-2 sentence summary of what data the pipeline is for and what it does -->
 **nf-core/pgdb** is a bioinformatics best-practise analysis pipeline for
```

Comment:
I think this sentence must be extended a bit and explain a bit clearer what the default pipeline call does.

Reply:
Thanks a lot, the README is now better.